spec/schemas/_core.msearch_template.yaml

```diff
@@ -25,7 +25,9 @@ components:
             The value is the variable value.
           type: object
           additionalProperties:
-            type: object
+            oneOf:
+              - type: string
+              - type: object
```

Comment:
What's an example where this is a string? The description says these are key/value pairs.

Reply:
Yes, it's a key-value pair. Here’s one example: [link](https://github.com/opensearch-project/opensearch-api-specification/blob/5666d684c6eaf0a96d93ce52e85a5de7164a8c05/tests/default/_core/msearch/template.yaml#L33).

Initially, it seems there was an expectation that the value would be in the form of an object, but I demonstrated with an example that it can also be a string.